### PR TITLE
Eitang mk 308 sandbox playbook modes

### PR DESCRIPTION
**Pull request by eitangol**

### CODE REVIEW

**Review by JonathanLande — DISMISSED** [review]

File: .gitpod.yml [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
-      command: |
-        open ${GITPOD_REPO_ROOT}/policies.yaml
-        cd /home/gitpod
-        POLICIES_DIR=${GITPOD_REPO_ROOT} docker compose --project-name sandbox up -d --wait --wait-timeout 60
-        for i in {1..3}; do curl --silent --output /dev/null -H "Content-Type: application/json" -d '{"scheme": "http", "host": "provider:4578", "path": "/rate-limit/exponential-backoff", "method": "GET", "sleep_ms": 20}' http://localhost:7897/traffic; done
-        gp ports await 3000
-        gp preview $(gp url 3000)/d/d2c76c83-9968-4ef6-a6a5-6105e661f9bf/lunar-sandbox
+  - name: Sandbox Terminal
+    command: |
+      cp ${GITPOD_REPO_ROOT}/scenarios/${SANDBOX_SCENARIO} ${GITPOD_REPO_ROOT}/policies.yaml
```
Comment (ezean86) [inline .gitpod.yml:29]: we currently don't have scenarios, we don't want to add Strategy-Based Throttling scenario?

File: .gitpod.yml [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+      open ${GITPOD_REPO_ROOT}/policies.yaml
+      cd /home/gitpod
+      SANDBOX_SCENARIO=${SANDBOX_SCENARIO} POLICIES_DIR=${GITPOD_REPO_ROOT} docker compose --project-name sandbox up -d --wait --wait-timeout 60
+      for i in {1..3}; do curl --silent --output /dev/null -H "Content-Type: application/json" -d '{"scheme": "http", "host": "provider:4578", "path": "/rate-limit/exponential-backoff", "method": "GET", "sleep_ms": 20}' http://localhost:7897/traffic; done
```
Comment (ezean86) [inline .gitpod.yml:33]: for i in {1..5}; do curl --silent --output /dev/null -H "Content-Type: application/json" -d '{"scheme": "http", "host": "provider:4578", "path": "/rate-limit/exponential-backoff", "method": "GET", "sleep_ms": 30}' http://localhost:7897/traffic; done

@JonathanLande ?

Comment (JonathanLande) [reply .gitpod.yml:33]: You're right, I will do this in a separate PR (I missed it until now that I need to change it here)

**Review by ezean86 — DISMISSED** [review]
👍

File: .gitpod.yml (hunk 2 above)
Comment (eitangol) [reply .gitpod.yml:33]: @ezean86, this was already like this.. didn׳t understand your comment

**Review by JonathanLande — APPROVED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (eitangol) [reply .gitpod.yml:29]: this is a scenario

**Pull request merged**